using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md

```diff
@@ -0,0 +1,417 @@
+  ///
+  /// - Note: The returned XDM data also contains the `eventType` for the Experience Event with value `click`.
+  /// - Returns A dictionary containing XDM data for the propositon interactions.
+  func generateTapInteractionXdm() -> [String: Any] {...}
+}
+```
+```swift
+public extension Proposition {
+    /// Creates a dictionary containing XDM formatted data for `Experience Event - Proposition Reference` field group from the given proposition.
+    ///
+    /// The Edge `sendEvent(experienceEvent:_:)` API can be used to dispatch this data in an Experience Event along with any additional XDM, free-form data, or override dataset identifier.
+    ///
+    /// - Note: The returned XDM data does not contain an `eventType` for the Experience Event.
+    /// - Returns A dictionary containing XDM data for the propositon reference.
+    func generateReferenceXdm() -> [String: Any] {...}
+}
+```
+{% endtab %}
+{% endtabs %}
+
+The Edge `sendEvent` API can then be used to send this tracking XDM data along with any additional XDM and freeform data to the Experience Edge network. Additionally, an override dataset can also be specified for tracking data. For more details, see [Edge - sendEvent API](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension/edge-network-api-reference#sendevent).
+
+{% tabs %}
+{% tab title="iOS" %}
+#### Swift
+
+```swift
+// When a proposition is retrieved using getPropositions API or onUpdatePropositions API callback 
+// and the corresponding offer is displayed, invoke method on Offer object to get the XDM data.
+
+let displayInteractionXdm = offer.generateDisplayInteractionXdm() // Offer display tracking XDM
+let additionalData: [String: Any] = ["someDataKey": "someDataValue"]
+
+let experienceEvent = ExperienceEvent(xdm: displayInteractionXdm, data: additionalData)
+Edge.sendEvent(experienceEvent)
+```
+
+#### Objective-C
+
+```objc
+// When a proposition is retrieved using getPropositions API or onUpdatePropositions API callback 
+// and the corresponding offer is displayed, invoke method on Offer object to get the XDM data.
+
+NSDictionary* displayInteractionXdm = [offer generateDisplayInteractionXdm];
+NSDictionary* additionalData = @{@"someDataKey": @"someDataValue"};
+
+AEPExperienceEvent* experienceEvent = [[AEPExperienceEvent alloc] initWithXdm:displayInteractionXdm data:additionalData datasetIdentifier:nil];
+[AEPMobileEdge sendExperienceEvent:event completion:nil];
+```
+{% endtab %}
+{% endtabs %}
+
+## Configuration keys
+
+To update the SDK configuration programmatically, use the following information to change the Optimize extension configuration values. For more information, see [Programmatic updates to Configuration](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration/configuration-api-reference#updateconfiguration).
```

Comment:
"For more information, see the [programmatic updates to Configuration tutorial].".

Reply:
Changed to [programmatic updates to Configuration guide]